### ISSUE TITLE
False positive with g_param_spec_static_strings rule

### ISSUE THREAD

**Issue by ptomato** [bilelmoussaoui/gobject-lint#62]
I have this code:
```c
    g_object_class_install_property(
        gobject_class, PROP_G_INTERFACE_INFO,
        g_param_spec_boxed(
            "g-interface-info", "Interface Info",
            "A DBusInterfaceInfo representing the exported object",
            G_TYPE_DBUS_INTERFACE_INFO,
            (GParamFlags)(G_PARAM_STATIC_STRINGS | G_PARAM_WRITABLE |
                          G_PARAM_CONSTRUCT_ONLY)));
```
This is flagged as needing `G_PARAM_STATIC_STRINGS`.

**Comment by bilelmoussaoui** [comment]
Is the casting here really useful?

**Comment by ptomato** [comment]
I mean ... no ... but it's still a false positive 😛
